recipes/libestools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "libestools" %}
+{% set version = "2.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/festvox/speech_tools/archive/63ff01938f81443f5a294bc5f9ea6ac6ab38f6b0.tar.gz
+  sha256: 8781ab15ce13fcddcb7df8fd23755a5cb6e4ab7d1a9bdbd741128b990b425496
+
+build:
+  skip: True  # [not linux]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - ncurses
+  run:
+    - _openmp_mutex
```

Comment:
hmm, do you really need thsi dependency? I don't think you are using `openmp` (I believe thsi would be automatically added when using https://conda-forge.org/docs/maintainer/knowledge_base.html#message-passing-interface-mpi)

Reply:
There was a CI error without it...

Reply:
I really really don't think this is necessary. I would love to see that CI error

Reply:
This was the output of my local test. I can try to remove this line in this PR, too. Just let me know based on this log:

```
   INFO: sysroot: '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/' files: '['/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/zone.tab', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/tzdata.zi', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/Zulu', '/home/conda/staged-recipes/build_artifacts/libestools_1674923795129/_build_env/x86_64-conda-linux-gnu/sysroot/usr/share/zoneinfo/right/WET']'
   INFO (libestools,lib/libestools.so.2.5): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so.2.5): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so.2.5): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5.1): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5.1): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so.2.5.1): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestbase.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestbase.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libeststring.so.2.5.1): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so): lib/libestbase.so.2.5 found in this package
   INFO (libestools,lib/libestools.so): lib/libeststring.so.2.5 found in this package
   INFO (libestools,lib/libestools.so): Needed DSO lib/libtinfo.so.6 found in conda-forge::ncurses-6.3-h27087fc_1
   INFO (libestools,lib/libestools.so): Needed DSO lib/libstdc++.so.6 found in conda-forge::libstdcxx-ng-12.2.0-h46fd767_19
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libm.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
  ERROR (libestools,lib/libestools.so): Needed DSO lib/libgomp.so.1 found in ['_openmp_mutex']
  ERROR (libestools,lib/libestools.so): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
   INFO (libestools,lib/libestools.so): Needed DSO lib/libgcc_s.so.1 found in conda-forge::libgcc-ng-12.2.0-h65d4601_19
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libpthread.so.0 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
   INFO (libestools,lib/libestools.so): Needed DSO x86_64-conda-linux-gnu/sysroot/lib64/libc.so.6 found in CDT/compiler package conda-forge::sysroot_linux-64-2.12-he073ed8_15
Traceback (most recent call last):
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 220, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 118, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "/home/conda/staged-recipes-copy/.ci_support/build_all.py", line 176, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 180, in build
    return build_tree(
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3097, in build_tree
    packages_from_this = build(metadata, stats,
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2369, in build
    newly_built_packages = bundlers[pkg_type](output_d, m, env, stats)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1658, in bundle_conda
    files = post_process_files(metadata, initial_files)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 1504, in post_process_files
    post_build(m, new_files, build_python=python)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1319, in post_build
    check_overlinking(m, files, host_prefix)
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1225, in check_overlinking
    return check_overlinking_impl(m.get_value('package/name'),
  File "/opt/conda/lib/python3.10/site-packages/conda_build/post.py", line 1204, in check_overlinking_impl
    raise OverLinkingError(overlinking_errors)
conda_build.exceptions.OverLinkingError: overlinking check failed 
["  ERROR (libestools,lib/libestools.so.2.5): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libestools,lib/libestools.so.2.5.1): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)", "  ERROR (libestools,lib/libestools.so): .. but ['_openmp_mutex'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)"]
Traceback (most recent call last):
  File "build-locally.py", line 101, in <module>
    main()
  File "build-locally.py", line 95, in main
    run_docker_build(ns)
  File "build-locally.py", line 33, in run_docker_build
    subprocess.check_call([script])
  File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['.scripts/run_docker_build.sh']' returned non-zero exit status 1.
```